rke/k8s_rke_system_images.go

```diff
@@ -6086,6 +6218,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.18.0",
 		},
+		// Enabled out of band post v2.5.11
```

Comment:
1.21 was introduced in v2.6.0, so this should be `//Enabled out of band post v2.6.2`

Reply:
updated